test/test_helper.rb

```diff
@@ -0,0 +1,13 @@
+ENV['RAILS_ENV'] ||= 'test'
+require_relative "../config/environment"
+require "rails/test_help"
+
+class ActiveSupport::TestCase
```

Comment:
If RSpec is installed, do you need Minitest?

Reply:
@unijewski I don't think we need Minitest. If we want to remove Minitest, do we need to remove these files only?
test/application_system_test_case.rb

Reply:
If so, how about deleting the whole `test` directory? If you stick to RSpec, then it shouldn't be needed.